### PR TITLE
Previewing SVG files works

### PR DESCRIPTION
**Pull request by fesave**
## Related Issues
App: #1033 

- [x] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt [hunk 1]
```diff
@@ -347,6 +358,7 @@ class PreviewImageFragment : FileFragment() {
         private const val ARG_FILE = "FILE"
         private const val ARG_ACCOUNT = "ACCOUNT"
         private const val ARG_IGNORE_FIRST = "IGNORE_FIRST"
+        private const val SVG_MIMETYPE = "image/svg+xml"
```
Comment (abelgardep) [inline owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt:361]: I think that this mimetype could be moved to https://github.com/owncloud/android/blob/master/owncloudDomain/src/main/java/com/owncloud/android/domain/files/MimeTypeConstants.kt to keep the mimetypes together. What do u think?

File: owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt [hunk 2]
```diff
@@ -324,18 +327,26 @@ class PreviewImageFragment : FileFragment() {
...
+                    override fun onResourceReady(
+                        resource: Drawable?, model: Any, target: Target<Drawable?>,
+                        dataSource: DataSource, isFirstResource: Boolean
+                    ): Boolean {
+                        Timber.d("Loading image %s", file.fileName)
+                        binding.progressWheel.isVisible = false
+                        return false
+                    }
+                })
+                .into(binding.photoView)
```
Comment (abelgardep) [inline owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt:339]: Is this properly formatted?

Comment (abelgardep) [inline owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt:339]: Is this properly formatted?

File: owncloudApp/build.gradle [hunk 3]
```diff
@@ -26,6 +26,7 @@ dependencies {
 
     // Image loading
     implementation 'com.github.bumptech.glide:glide:4.12.0'
+    implementation 'com.github.corouteam:GlideToVectorYou:v2.0.0'
```
Comment (abelgardep) [inline owncloudApp/build.gradle:29]: Is this dependency needed?

Comment (abelgardep) [inline owncloudApp/build.gradle:29]: Is this dependency needed?

**Review by abelgardep — CHANGES_REQUESTED** [review]
Some suggestions here @fesave

File: owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt (hunk 1 above)
Comment (fesave) [reply owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt:361]: I was unaware of the existence of this class, perfect. Changed.

File: owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt (hunk 2 above)
Comment (fesave) [reply owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt:339]: Reformated code again.

Comment (fesave) [reply owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt:339]: Repeated 😮

File: owncloudApp/build.gradle (hunk 3 above)
Comment (fesave) [reply owncloudApp/build.gradle:29]: The library is based on Glide and offers the same functionalities + SVG support. Just by adding the dependency, SVG support works. Maybe the question is... Do we need the original glide dependency if we use this new dependency or can they coexist together? What do you think?

Comment (fesave) [reply owncloudApp/build.gradle:29]: Why do you repeat the comments? Is it a bug? haha 😆

**Review by abelgardep — APPROVED** [review]

**Comment by jesmrec** [comment]
SVG preview works fine. Approved

**Pull request merged**